region/maridia/inner-yellow.json

```diff
@@ -1156,6 +1315,18 @@
           "locks": null
         }
       ],
+      "enemies": [
+        {
+          "name": "Pink Space Pirate (standing)",
+          "quantity": 4,
+          "homeNodes": [ 2 ]
```

Comment:
I don't suppose it matters much, but one of them is surely positioned at node 1.

Reply:
Since the door is grey I don't see it having an impact no.